docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
- const litNodeClient = new LitNodeClient({
-  litNetwork: "datil-dev",
-  debug: true,
-  storageProvider: {
-    provider: new LocalStorage('./storage.test.db'),
-  }
- });
- ```
-:::
-
-The `getSessionSigs()` function will try to create a session key for you and store it in local storage. You can also generate the session key yourself using `generateSessionKeyPair()` function and store it however you like. You can then pass the generated session key to `getSessionSigs()` as the `sessionKey` param.
-
-In the example above, we construct a SIWE message manually, but you can use the `checkAndSignAuthMessage` when in a browser context.
+**Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
 ## Resources You Can Request
 
-You can pass an array of "resource ability requests" to the `getSessionSigs()` function, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
+You can pass an array of `resourceAbilityRequests` to the above functions, which will be presented to the user in the SIWE message - read more [here](resources-and-abilities) about Lit resources and abilities. The resources and abilities requested by the session key must be narrower or equal to the capabilities granted to it per the session capability object specified in the inner `AuthSig`. 
 
-When session capability objects are omitted from the `getSessionSigs()` function call, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
+When session capability objects are omitted from functions generating session signatures, the SDK will generate a session capability object with **wildcard permissions against all of the resources in that category by default**, i.e. ability to perform operations against all access control conditions. Read more [here](capability-objects) about how to create custom session capability objects.
 
 ## Clearing Local Storage
 
 If you want to clear the session key stored in the browser local storage, you can call the [`disconnectWeb3` method](https://js-sdk.litprotocol.com/functions/auth_browser_src.ethConnect.disconnectWeb3.html).
 
+## Summary
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getSessionSigs). 
+
+After executing the example implementation above, you will have generated and stored your session public and secret keys, and the `AuthSig` signed for the session signatures. 
```

Comment:
```suggestion
After executing the example implementation above, you will have generated and stored session signatures that allow you to request to decrypt data that you satisfy the Access Control Conditions for.

You can use these session signatures in a decryption request [like so](../../../access-control/quick-start#decryption).
```

Reply:
What about:

After executing the example implementation above, you will have generated and stored session signatures that allow requests to decrypt data that match your Access Control Conditions.

Reply:
```
After executing the example implementation above, you will have generated and stored session signatures that allow requests to decrypt data that you satisfy the Access Control Conditions for.
```

Were not matching the control conditions, they're a set of constraints that we're satisfying